### ISSUE TITLE
[enhancement] instrumentation: naming of internal metrics sent to Graphite

### ISSUE THREAD

**Issue by daks** [graphite-ng/carbon-relay-ng#334]
Hello,

you can/must send own internal metrics (instrumentation) to a Graphite server. The time interval and address of this server is configurable but not the name used to send these metrics, which is `service_is_carbon-relay-ng`.

Do you think it could be possible to add a variable to set thoses metrics names?

**Comment by Dieterbe** [comment]
Yes. But probably we should redo the entire metric naming system to adopt the new graphite tags mechanism. (With configurable prefix ) .
